taxonomy-editor-frontend/src/pages/editentry/AccumulateAllComponents.jsx

```diff
@@ -145,14 +158,18 @@ const AccumulateAllComponents = ({ id, taxonomyName, branchName }) => {
           />{" "}
         </>
       )}
-      {/* Button for submitting edits */}
-      <Button
-        variant="contained"
-        onClick={handleSubmit}
-        sx={{ ml: 4, mt: 2, width: 130, mb: 2 }}
-      >
-        Submit
-      </Button>
+      {/* Fab for submitting edits */}
+      {changesMade && (
+        <Fab
```

Comment:
could we add some background color for the Fab, just so that the button is not floating on top of the content?